Modules/Core/Common/src/itkImageSourceCommon.cxx

```diff
@@ -25,9 +25,7 @@ namespace itk
 
 struct ImageSourceCommonGlobals
 {
-  ImageSourceCommonGlobals() { m_GlobalDefaultSplitter = ImageRegionSplitterSlowDimension::New().GetPointer(); };
-
-  ImageRegionSplitterBase::Pointer m_GlobalDefaultSplitter;
+  ImageRegionSplitterBase::Pointer m_GlobalDefaultSplitter{ ImageRegionSplitterSlowDimension::New().GetPointer() };
```

Comment:
The original default-constructor of `ImageSourceCommonGlobals` does (implicitly) default-construct the smart pointer `m_GlobalDefaultSplitter` (_null_-initializing its pointer), and then assigns a new value from `ImageRegionSplitterSlowDimension::New()` to the pointer. My proposal does instead directly initialize `m_GlobalDefaultSplitter` to the new value. Semantically both should have the same effect. Unless some other thread tries to access the `ImageSourceCommonGlobals` object while it is still "under construction". Is that happening currently?